app/views/shared/_students.html.erb

```diff
@@ -0,0 +1,46 @@
+<%
+  overview_path = @organisation ? :organisation_cohort_path : :cohort_path
```

Comment:
Let's not compute the paths here. Send it as argument to the partial from the calling site.